src/run_ceremony.nix

```diff
@@ -0,0 +1,148 @@
+{ config, all_scripts, pkgs, ... }:
+let
+  scripts_to_run_in_order = builtins.map (name: all_scripts.custom_scripts."${name}") config.orca.actions_in_order;
+  orca_protocol = all_scripts.orca_scripts.orca_user;
+  computeCVault = pkgs.lib.getExe all_scripts.orca_scripts.orca_user.compute_c_vault;
+  count_tokens = pkgs.lib.getExe all_scripts.orca_scripts.orca_user.count-tokens;
+  inherit (config.environment.variables) OUTPUT_FOLDER;
+  inherit (config.orca) latest_cvault rotate_keys;
+  expect_initialized = latest_cvault != null;
```

Comment:
This comparison is done en second time below. I'm wondering if we can't factorize.
Something like:
```
vault_is_pristine = latest_cvault == null;
```
and a bit below:
```
    ${ if vault_is_pristine then ''
    echo "This is the first time O.R.CA is started so no Cvault needs to be checked"
    '' else ''
    (...)
    ''
```

Reply:
I factorized, the renaming could be discussed.